radicle-cli/tests/commands.rs

```diff
@@ -190,7 +190,6 @@ fn rad_id_rebase() {
 }
 
 #[test]
-#[ignore]
```

Comment:
We should keep this for now, otherwise tests might fail on developer machines.

Reply:
Sorry this is a mistake while I was fix the git history